charts/console/examples/console-enterprise.yaml

```
@@ -0,0 +1,79 @@
+image:
+  tag: master-8fcce39
```

```suggestion
  tag: v2.1.0
```

Because we switched to branching out push releases into a separate docker repository than the repository I can imagine documenting two cases here:

1. Image.tag that refers to a tagged version like `v2.1.0`
2. image.tag + image.repository which refer to our `console-unstable` repository that is proxied via docker.redpanda.com

---

Again, no material changes in this migration PR.